src/Servers/Kestrel/Core/src/Internal/Http2/Http2Stream.cs

```diff
@@ -57,7 +57,7 @@ public Http2Stream(Http2StreamContext context)
                 this,
                 context.ServiceContext.Log);
 
-            RequestBodyPipe = CreateRequestBodyPipe(context.ServerPeerSettings.InitialWindowSize);
+            RequestBodyPipe = CreateRequestBodyPipe(context.ServerPeerSettings.InitialWindowSize * 10000); // TODO Once corefx is updated remove this
```

Comment:
File an issue, and make the pipe Pause/ResumeWriterThreashold long.MaxValue in the meantime.

Reply:
We should get updated dependencies before EOD. Anyways, did we decide what the default will be for the thresholds?

Reply:
The default wouldn't need to change with the new design.